### ISSUE TITLE
getVersion() returns CloudSdkOutOfDateException for Cloud SDK installed by pacakge manager

### ISSUE THREAD

**Issue by chanseokoh** [GoogleCloudPlatform/appengine-plugins-core#298]
Verified on Ubuntu. Other Linux distros might be affected too.

When installed through apt-get ([quickstart instructions here](https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu)), the Cloud SDK home is `/usr/lib/google-cloud-sdk`. The directory does not contain the file `VERSION`, so `getVersion()` always returns `CloudSdkOutOfDateException`.

```
chanseok@M32AD-R14:~$ ls -l /usr/lib/google-cloud-sdk/
total 140
drwxr-xr-x 3 root root   4096 Jan 15 10:55 bin/
-rw-r--r-- 1 root root   2555 Jan 12 18:19 completion.bash.inc
drwxr-xr-x 6 root root   4096 Jan 15 10:55 lib/
drwxr-xr-x 5 root root   4096 Jan 15 10:55 platform/
-rw-r--r-- 1 root root     39 Jan 12 18:21 properties
-rw-r--r-- 1 root root    673 Jan 12 18:19 README
-rw-r--r-- 1 root root 118151 Jan 12 18:19 RELEASE_NOTES
chanseok@M32AD-R14:~$
```

A workaround would be to download a versioned Cloud SDK archive instead of using package managers. (Such an archive does have the `VERSION` file.)

**Comment by elharo** [comment]
Good detective work. Please file a bug against the Cloud SDK on this. I don't think this is our bug.

**Comment by chanseokoh** [comment]
> A workaround would be to download a versioned Cloud SDK archive instead of using package managers.

When using the workaround, it is advised to add the `bin` directory of the unzipped home at the front of `PATH` to avoid executing `/usr/bin/gcloud` accidentally.

**Comment by chanseokoh** [comment]
Filed a bug against the Cloud SDK: 34310195.

On our side, we will need to increase [`MINIMUM_VERSION`](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/master/src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java#L416) to the first version in which Cloud SDK ships `VERSION`.

**Comment by elharo** [comment]
Actually I don't think we do. We should update the docs, but if someone has installed an older version such as 133 that does not have this problem it should still work. And we'll still show a CloudSdkOutOfDate message to anyone who does have the problem.

**Comment by chanseokoh** [comment]
They say that this is fixed since gcloud "141.0.0".

**Comment by loosebazooka** [comment]
That's really strange, how do they check it?
